src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
+    }
```

Comment:
To maintain consistency in object initialization, consider providing a constructor that initializes all the fields of the Book class, including the 'id'. This will allow you to create Book objects in a consistent manner using either the constructor or setters, but not both. According to the checklist, it's not consistent to use both ways of initialization.